### ISSUE TITLE
Copy over keyboard navigation plugin code and remove plugin dependency

### ISSUE THREAD

**Issue by rachel-fenichel** [rachel-fenichel/blockly-keyboard-experimentation#35]
When I started this project I extended the classes in the keyboard-navigation plugin, for simplicity. But that also means that the code is split across two repositories, which makes it hard to modify. 

The relevant parts of the keyboard navigation source should be copied in, and the dependency should be deleted.
